tools/cxbe/Xbe.cpp

```diff
@@ -686,6 +697,36 @@ Xbe::Xbe(class Exe *x_Exe, const char *x_szTitle, bool x_bRetail)
             }
 
             hwc = hwc_secn;
+            szBuffer = m_HeaderEx + hwc - (m_Header.dwBaseAddr + sizeof(m_Header));
+        }
+
+        // Write library versions
+        {
+            m_LibraryVersion = new LibraryVersion[m_Header.dwLibraryVersions];
+
+            for(uint32_t v = 0; v < m_Header.dwLibraryVersions; ++v)
```

Comment:
I believe this codebase uses a different type (probably `int`, `unsigned int` or `uint32` [without the `_t`]). Please check surrounding code.